packages/airnode-node/src/evm/gas-prices.ts

```diff
@@ -2,46 +2,63 @@ import { BigNumber, ethers } from 'ethers';
 import { go } from '../utils/promise-utils';
 import * as logger from '../logger';
 import { BASE_FEE_MULTIPLIER, DEFAULT_RETRY_TIMEOUT_MS, PRIORITY_FEE } from '../constants';
-import { GasTarget, LogsData, PendingLog } from '../types';
+import { ChainOptions, GasTarget, LogsData, PendingLog, PriorityFee } from '../types';
 
-interface FetchOptions {
+export interface FetchOptions {
   readonly provider: ethers.providers.JsonRpcProvider;
+  readonly chainOptions: ChainOptions;
 }
 
-export const getGasPrice = async (options: FetchOptions): Promise<LogsData<GasTarget | null>> => {
+export const parsePriorityFee = ({ value, unit }: PriorityFee) => ethers.utils.parseUnits(value, unit ? unit : 'wei');
```

Comment:
```suggestion
export const parsePriorityFee = ({ value, unit }: PriorityFee) => ethers.utils.parseUnits(value, unit ??  'wei');
```

Reply:
!! I didn't know you could do that - awesome :)